.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -19,44 +19,54 @@ Feel free to remove this section before creating this issue.
 -->
 
 **Describe the bug**
+
 A clear and concise description of what the bug is.
 
 **To reproduce**
+
 Steps to reproduce the behavior:
+
 1. Go to '...'
 2. Click on '....'
 3. Scroll down to '....'
 4. See error
 
 **Expected behavior**
+
 A clear and concise description of what you expected to happen.
 
 **Screenshots**
+
 If applicable, add screenshots to help explain your problem.
 
 **Additional context**
-Add any other context about the problem here.
+
+Paste the output of `yarn build:report`, and any other context about the problem, here.
 
 **Possible solutions**
+
 Add any ideas about possible solutions to the problem here.
 
 **Please complete the following device information:**
- - Device [e.g. iPhone6, PC, Mac, Pixel3]:
- - OS [e.g. iOS8.1, Windows 10]:
- - Browser [e.g. Chrome, Safari]:
- - Browser Version [e.g. 22]:
- - Magento Version:
- - PWA Studio Version:
- - NPM version `npm -v`:
- - Node Version `node -v`:
+
+-   Device [e.g. iPhone6, PC, Mac, Pixel3]:
```

Comment:
Instead of asking them to fill out all this info, it makes their life easier if they can simply paste the output of the `yarn build:report` command. Might increase adoption.

Reply:
I added a bit of text to the "additional context" section. The rest of this was just auto-formatting.

Reply:
True, I understand this is what we had before, I am trying to see how we can increase the adoption of this amazing tool. I don't think we often see issues with all this info provided. If we can simply make the tool copy the output to the clipboard and they can paste it here, it will make ours and their life easier.

It is a suggestion, I am fine with the current approach. Ill leave it to you.

Reply:
I'll remove the stuff that the generator should include.